src/models/UserMapper.php

```diff
@@ -174,6 +196,10 @@ public function transformResults($results, $verbose)
                     . $row['ID'] . '/hosted/';
                 $list[$key]['talk_comments_uri'] = $base . '/' . $version . '/users/'
                     . $row['ID'] . '/talk_comments/';
+                
+                if ($verbose && isset($this->_request->user_id)) {
```

Comment:
If you're not logged in, then this field won't be set.  This function could be used by both logged in and anonymous users I think, but only if you are logged in and have rights would we want to add this field.